docs/developer/api/serialization.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Serialization groups
+---
+
+*Roadiz* CMS uses `symfony/serializer` to perform JSON serialization
+over any objects, especially *Doctrine* entities.
+
+id
+> Serialize every entity `id`.
```

Comment:
I think this should be better converted to Markdown array syntax

```md
|  Group  |  Description |
| --- | --- |
| `id` | Serialize every entity `id`. |
| `timestamps` | Serialize every date-timed entity `createdAt` and `updatedAt` fields. |
...
```

to output somthing like this:

|  Group  |  Description |
| --- | --- |
| `id` | Serialize every entity `id`. |
| `timestamps` | Serialize every date-timed entity `createdAt` and `updatedAt` fields. |